server/block/sponge.go

```diff
@@ -15,6 +15,8 @@ type Sponge struct {
 
 	// Wet specifies whether the dry or the wet variant of the block is used.
 	Wet bool
+	// Particles specifies whether the block needs particles, only used when a wet sponge is placed in the nether.
+	Particles bool
```

Comment:
Why a field?

Reply:
will change it to a regular variable in my next commit